desktop-core/src/main/java/configs/CoreConfig.java

```diff
@@ -0,0 +1,10 @@
+package configs;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan("/guiApp")
```

Comment:
Annotation ComponentScan should containing base package name for search components, see [Spring doc](http://docs.spring.io/spring-javaconfig/docs/1.0.0.M4/reference/html/ch06s02.html).